packages/attestation-service/src/models/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
```

Comment:
This looks like a compiled file. Why is this here?

Reply:
It's a generated file by sequelize scaffold. It looks like it is supposed to give unified access to all the models, but since we are not using that, I will remove the file